phalcon/mvc/model/query/builderinterface.zep

```diff
@@ -202,6 +202,25 @@ interface BuilderInterface
 	/**
 	 * Sets an OFFSET clause
 	 */
-	 public function offset(int offset) -> <BuilderInterface>;
+	public function offset(int offset) -> <BuilderInterface>;
```

Comment:
Did you use spaces instead of tabs?

Reply:
I hope not. It should have been picked up by `phpcs` if I did. If I don't fix it in my next PR assume it's a non-issue.

Reply:
@CameronHall phpcs know nothing about *.zep files

Reply:
RIP. I'll fix in my next PR which will be addressing #13552.

Reply:
@sergeyklay looks like I removed the space in the commit so it already existed I suppose 👍

Reply:
@CameronHall :+1:  Thank you